funding/funding brief

```diff
@@ -0,0 +1,14 @@
+Berichtgeving over het veranderende klimaat is doorgaans pessimistisch en ingewikkeld. Dit zorgt ervoor dat lezers een aversie ervaren jegens het onderwerp. "Het kost mij teveel om het uit te zoeken en wat ik doe heeft toch niet zo'n groot effect". Het maatschappelijk pessimisme heeft niet alleen een effect op het gedrag van individuen maar  indirect ook op dat van overheden en organisaties.
```

Comment:
Misschien ook toevoegen over dat het maken van de nieuwsbrief ons, de redactie, heeft veranderd en dat wij op een gheel andere manier naar de toekomst kijken dan voor dat we de nieuwsbrief maakten.

Inspiratie: [Medium post HWHBSTPTW](https://medium.com/new-atoms-beta/how-we-have-been-saving-the-planet-this-week-ca0219ba557f#.dk81m3e8v)